api/cloudbuild.yaml

```diff
@@ -37,12 +37,20 @@ steps:
       - 'MINIO_SECRET_KEY=$_MINIO_SECRET_KEY'
       - 'MINIO_BUCKET_NAME=$_MINIO_BUCKET_NAME'
 
-  - name: gcr.io/kaniko-project/executor:debug
+  - name: 'gcr.io/cloud-builders/docker'
```

Comment:
need the quotes? or is that just Prettier?

Reply:
There isn't really a good reason for that. I'll get rid of it.